modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/RequiredInformation.jsx

```diff
@@ -166,16 +177,25 @@ const RequiredInformation = ({
 			startDate: infoSelectedKey?.selectedSubscription.startDate,
 		};
 
+		const subscriptionKey = async (id) => {
+			await putSubscriptionInKey(provisioningServerAPI, id, sessionId);
+		};
+
 		if (infoSelectedKey.hasNotPermanentLicence) {
-			await createNewGenerateKey(
+			setIsLoadingUserInvitation(true);
+
+			const results = await createNewGenerateKey(
 				accountKey,
 				provisioningServerAPI,
 				sessionId,
 				licenseKey
 			);
-		}
-		else {
-			await Promise.all(
+			await subscriptionKey(results[0].items.id);
+			setIsLoadingUserInvitation(false);
```

Comment:
1. Spaces
2. Is this right `results[0].items.id` ?  or `results.items[0].id` ?

```suggestion

			await saveSubscriptionKey(results[0].items.id);
			
			setIsLoadingUserInvitation(false);
```

Reply:
The correct is this: results[0].items[0].id